elementary/exceptions/exceptions.py

```diff
@@ -41,19 +42,30 @@ def __init__(
         self,
         err: subprocess.CalledProcessError,
         base_command_args: List[str],
-        err_msg: Optional[str],
-    ):
+        err_msg: Optional[str] = None,
+        logs: Optional[List[DbtLog]] = None,
+    ):  
         msg = "Failed to run dbt command."
+        if logs and not err_msg:
+            err_msg = '\n'.join([log.msg for log in logs if log.level == 'error'])
         if err_msg:
             msg = f"{msg}\n{err_msg}"
-
         super().__init__(msg)
 
         # Command args sent to _run_command (without additional user-specific args it as such as projects / profiles
         # dir)
         self.proc_err = err
         self.base_command_args = base_command_args
         self.return_code = err.returncode
+        self.logs = logs
+    
+    @classmethod
+    def from_process_error(cls, err: subprocess.CalledProcessError, base_command_args: List[str], err_msg: Optional[str] = None) -> 'DbtCommandError':
+        if err.output is None:
+            return cls(err, base_command_args, err_msg, None)
```

Comment:
Can you be explicit and pass `logs=None`?

Reply:
Sure